<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabCreate</class>
 <widget class="QWidget" name="TabCreate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>197</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>274</width>
    <height>197</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0" rowminimumheight="0,0,0,0,0">
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Random Bound(min,max)</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QSpinBox" name="minRandomBox"/>
         </item>
         <item>
          <widget class="QSpinBox" name="maxRandomBox"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>number of columns</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QRadioButton" name="xlsxButton">
         <property name="text">
          <string>xlsx generation</string>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinColumns"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>number of rows</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QRadioButton" name="csvButton">
         <property name="text">
          <string>csv generation</string>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinRows"/>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="randomButton">
         <property name="text">
          <string>Random Initialisation</string>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QRadioButton" name="noButton">
         <property name="text">
          <string>No Initialisation</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TabCreate</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
